lib/partitions_validator_utils.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package partitions_validator_utils;
+use Exporter 'import';
+use strict;
+use warnings;
+use scheduler 'get_test_suite_data';
+use testapi;
+use Test::Assert ':all';
+our @EXPORT = qw(validate_partition_table);
+
+
+sub validate_partition_table {
+    my ($args) = @_;
+
+    return if check_var('BACKEND', 's390x');    # fdisk output does not show partition table for dasd
+
+    record_info("Check $args->{expected_table_type}", "Verify if partition table type is $args->{expected_table_type}");
+    my @fdisk_output = split(/\n/, script_output("fdisk -l $args->{device}"));
+    my $table_type;
+    foreach (@fdisk_output) {
+        if ($_ =~ /Disklabel\stype: (.+)$/) { $table_type = $1 }
```

Comment:
notation for capture groups is `(?<name>...)` and you use it with `$+{name}`

Reply:
Thanks for the tip. I'll keep it as it is for now though.